src/Core/Framework/Demodata/DemodataService.php

```diff
@@ -20,6 +20,8 @@
 #[Package('framework')]
 class DemodataService
 {
+    public const DEMODATA_CUSTOM_FIELDS_KEY = 'shopware.demodata';
```

Comment:
I would prefer snake case, since a dot can represent a path and it always has to be quoted